### DISCUSSION TITLE
Which Dagger (Cloak) extensions should we all create next?

### DISCUSSION THREAD

**Discussion by jpadams** [dagger/dagger#3216] [General]
"[Extensions](https://github.com/dagger/dagger/blob/cloak/docs/unxpq-introduction.md#extensions)" are how we add capabilities/integrations to Dagger in the next version (codename Cloak) .

[[learn more](https://github.com/dagger/dagger/blob/cloak/docs/unxpq-introduction.md)]

Examples today beyond core capabilities include Netlify, Vercel, Yarn, Terraform, Rails, etc

- Which extensions should we prioritize first? Why?
- Given your experience, which ones could you help to create or improve?
- Which Dagger v0.2 extensions (CUE) should be ported over to multi-language Cloak? [[stable](https://github.com/dagger/dagger/tree/main/pkg/universe.dagger.io)] [[alpha](https://github.com/dagger/dagger/tree/main/pkg/universe.dagger.io/alpha)]

Let us know below!

Please make one comment per proposed extension so folks can upvote them 👍

**Comment by jpadams** [comment]
Gradle

**Comment by jpadams** [comment]
Terraform

**Comment by jpadams** [comment]
Pulumi

**Comment by jpadams** [comment]
Go build/go-releaser/etc

**Comment by jpadams** [comment]
Rust/Cargo

**Comment by munichbughunter** [comment]
AWS Sam

**Comment by kpenfound** [comment]
Yarn/npm

**Comment by kpenfound** [comment]
rails/bundle

**Comment by kpenfound** [comment]
python/pip

**Comment by kpenfound** [comment]
dotnet

**Comment by kpenfound** [comment]
Hashicorp Vault

**Comment by kpenfound** [comment]
1password

**Comment by kpenfound** [comment]
AWS SSM

**Comment by slumbering** [comment]
nginx

**Comment by nipuna-perera** [comment]
Maven

**Comment by Alvise88** [comment]
Ansible

**Comment by magick93** [comment]
Supabase

**Comment by magick93** [comment]
Curl
